main/updatesettings.cfg

```diff
@@ -0,0 +1,10 @@
+writecfg "backupb4update"
```

Comment:
what's this file doing?

Reply:
It creates a backup cfg file before executing those commands.  Some of the added cfgs like this one exist so people can `exec updatesettings.cfg` and it execs commands that are considered standard now days, but if they want to go back a backup gets saved.